clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -8537,9 +8537,10 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     }
   }
 
-  if (IsCuda) {
+  if (IsCuda || (IsSYCLDevice && Triple.isNVPTX())) {
```

Comment:
nit: Please add some comments for the newly added code.

Reply:
Done, thanks